### DISCUSSION TITLE
CLOB/BLOB attributions

### DISCUSSION THREAD

**Discussion by JeremyDKellett** [USACE/cwms-data-api#858] [Ideas]
Is attribution like user_Id_uploaded and, especially, date_uploaded on a roadmap for additional development.

**Comment by MikeNeilson** [comment]
It wasn't, it is now no timeline yet. There doesn't appear any columns for that data at this time in the schema.
We technically have that information available but it would involve tracing things through the logs. user needs to not be exposed to anonymous users but that's not incredibly difficult to handle.
